visual-inspection/inspection-backend/inspection/explainer/explain.py

```diff
@@ -9,18 +9,20 @@
 from xaidemo.tracing import add_span_attributes, traced
 
 from .explainers.lime_ import lime_explanation
+from .explainers.visuallime_ import visuallime_explanation
 from ..model.model import get_model
 from ..model.predict import preprocess
 
 EXPLAINERS = {
-    "lime": lime_explanation
+    "lime": lime_explanation,
+    "visuallime": visuallime_explanation
 }
 
 
 @traced
 def generate_output_image(raw_image: np.ndarray,
                           size: Tuple[int, int]) -> bytes:
-    exp_image = Image.fromarray((255 * raw_image).astype(np.uint8))
+    exp_image = Image.fromarray(raw_image)
```

Comment:
Funktioniert das dann noch, wenn als Methode 'lime' ausgewählt wird?

Wir brauchen in jedem Fall End-to-End Backend-Tests sowohl für LIME als auch für VisuaLIME, d.h. eine POST-Anfrage mit "method: lime" und eine Anfrage mit "method: visualime".

Und dann müssen wir schauen, dass auch die Farben usw. noch stimmen, also das Bild nicht kaputt-konvertiert wird.

Reply:
Ja, das Funktioniert, ich habe lime_.py so umgeschrieben, dass das passt

Reply:
Prima, dann fehlen nur noch die Tests dafür :)